unit-tests/utilities/version/common.h

```diff
@@ -0,0 +1,7 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../../catch.h"
```

Comment:
I think `#include <unit-tests/test.h>` is bettwe, no?

Reply:
done